topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -0,0 +1,327 @@
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. Import data to OpenRefine
+> 2. Faceting
+> 3. Filtering
+> 4. Clustering
+> 5. API use
+> {:toc}
+>
+{: .agenda}
+
+# Retrieve data from GBIF "Biodiversity data mobilization course"
+
+## Get data
+
+
+> <hands-on-title> Data import and project creation </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “Openrefine tutorial”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 2. Import data file from GBIF {% cite GBIFtraining2021 %} "Biodiversity data mobilization course" accessible on [GBIF website](https://docs.gbif.org/course-data-mobilization/en/) on Use case 1 ["Herbarium Specimens"](https://docs.gbif.org/course-data-mobilization/en/exercise-3a-c.html) using file from Exercice 3C, at the bottom of the page
+>
+>    "Herbarium Specimens" data file as a csv file named "UC1-3c-open-refine.csv"
+>    ```
+>    https://docs.gbif.org/course-data-mobilization/exercise-data/UC1-3c-open-refine.csv
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>
+> 3. {% tool [Regex Find And Replace](toolshed.g2.bx.psu.edu/repos/galaxyp/regex_find_replace/regex1/1.0.3) %} with the following parameters:
+>    - {% icon param-file %} *"Select lines from"*: `UC1-3c-open-refine.csv`
+>    - In *"Check"*:
+>        - {% icon param-repeat %} *"Insert Check"*
+>            - *"Find Regex"*: `;`
+>            - *"Replacement"*: `\t`
+> 4. Change the datatype from "txt" to "tabular"
+> 
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+> 
+> 6. Check your output. All columns must be separated by a tabulation now. See example below.
+>
+> ![Output of Regex Find And Replace example](../../images/openrefine/1_openrefine.png "Output of Regex Find And Replace example")
+> 
+>
+> 7. Rename the datasets if needed, for example with "UC1-3c-open-refine-tabular" to keep orginal name and specify the tabular format.
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 
+>
+{: .hands_on}
+
+
+## Deploy an Openrefine instance and push your data in
+
+
+> <hands-on-title> Deploy an Openrefine instance and import data from your Galaxy history </hands-on-title>
+>
+> 1. Deploy an OpenRefine instance clicking Execute on the {% tool [OpenRefine tool](interactive_tool_openrefine) %} specifying "UC1-3c-open-refine-tabular" as input dataset
```

Comment:
Please use the GTN formatting for hands-on instructions:

```suggestion
> 1. {% tool [OpenRefine tool](interactive_tool_openrefine) %} with the following parameters:
>    - *"Input file in tabular format"*: `UC1-3c-open-refine-tabular`
```

Reply:
THANK YOU !